maplibre_gl/ios/maplibre_gl/Sources/maplibre_gl/MapLibreMapController.swift

```diff
@@ -261,6 +261,123 @@ class MapLibreMapController: NSObject, FlutterPlatformView, MLNMapViewDelegate,
         case "map#getTelemetryEnabled":
             let telemetryEnabled = UserDefaults.standard.bool(forKey: "MLNMapboxMetricsEnabled")
             result(telemetryEnabled)
+        case "map#setMaximumFps":
+            result(nil)
+        case "map#forceOnlineMode":
+            // Force online mode by ensuring network requests are enabled
+            // In MapLibre GL iOS, this is typically handled by the style and data sources
+            result(nil)
+        case "camera#ease":
+            guard let arguments = methodCall.arguments as? [String: Any] else { 
+                result(false)
+                return 
+            }
+            guard let cameraUpdate = arguments["cameraUpdate"] as? [Any] else { 
+                result(false)
+                return 
+            }
+            guard let camera = Convert.parseCameraUpdate(cameraUpdate: cameraUpdate, mapView: mapView) else { 
+                result(false)
+                return 
+            }
+
+            let completion = {
+                result(true)
+            }
+
+            if let duration = arguments["duration"] as? Double, duration > 0 {
+                let interval: TimeInterval = duration / 1000.0
+                mapView.setCamera(camera, withDuration: interval, animationTimingFunction: CAMediaTimingFunction(name: CAMediaTimingFunctionName.linear), completionHandler: completion)
```

Comment:
Use `CAMediaTimingFunctionName.easeInEaseOut` instead of `linear`, for the ease animation that we want
```suggestion
                mapView.setCamera(camera, withDuration: interval, animationTimingFunction: CAMediaTimingFunction(name: CAMediaTimingFunctionName.easeInEaseOut), completionHandler: completion)
```